src/Transition_Systems-Weak_Transition_Systems.thy

```diff
@@ -284,6 +284,30 @@ proof -
     by simp
 qed
 
+lemma weak_tau_transitions_up_to_weak_bisimilarity_rule:
```

Comment:
I think the lemma name should be changed.

For one, it shouldn’t contain `tau`, as `τ` is just the symbol we use for this certain action, while the name used for it throughout the transition system framework is `silent`.

Then there is also the issue that the current lemma name doesn’t mention the aspect that the silent transitions turn the two processes into each other (up to bisimilarity).

What about `mutual_silent_weak_transitions_up_to_bisimilarity`?

Reply:
Sounds good (actually, my first attempt used "`mutual_`" as the prefix :grin:).

Reply:
Fixed in f63b992834023eaa62d56e2669bff209ff6f856c, please verify.

Reply:
Looks good.